src/components/fields/AxesSelector.js

```diff
@@ -16,36 +16,45 @@ class AxesSelector extends Component {
   }
 
   render() {
-    const {axesTargetHandler, axesTarget, localize: _} = this.context;
+    const {
+      axesTargetHandler,
+      axesTarget,
+      fullLayout,
+      localize: _,
+    } = this.context;
     const {axesOptions} = this.props;
     const maxOptions = axesOptions.length > 4; // eslint-disable-line
 
-    if (maxOptions) {
-      return (
-        <Field {...this.props} label={_('Axis to Style')}>
-          <Dropdown
-            options={axesOptions.map(option => {
-              if (option.value !== 'allaxes') {
-                return {
+    const multipleSublots =
+      fullLayout &&
+      fullLayout._subplots &&
+      Object.values(fullLayout._subplots).some(s => s.length > 1);
```

Comment:
hmm, fullLayout._subplots is an object right
that contains subplotTypes as keys, and arrays as values, and we want to check that those arrays are of length greater than 1.

so isn't it more like:
`Object.values(fullLayout._subplots).some(s => fullLayout[s].length > 1);`

Reply:
Vera, it's Object.values(), not Object.keys()